src/task/Material.java

```diff
@@ -0,0 +1,32 @@
+package task;
+
+public class Material {
+
+    private final String NAME;
+    private final float DENSITY;
```

Comment:
```suggestion
    private final String name;
    private final float density;
```
это константы **экземпляра** класса.